Makefile

```diff
@@ -32,8 +32,10 @@ install-safety:
 
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
 	# 12 Dec 2023: 51668 is fixed with >= 2.0.0b1 of SQLAlchemy. Ongoing refactor to upgrade.
+	# 6 June 2024: 70624 vulnerability found with flask-cors version 4.0.0 
+	# 6 June 2024: 70612 vulnerability found with jinja2 version 3.1.3
```

Comment:
If a version exists that clears a vulnerability, please annotate the version here.

Reply:
We discussed this.